wordpress-mobile/WordPress-Android
# Feedback form: attachment pager

Fixes #21124 

This PR adds a reusable media pager to the feedback form which displays a preview of the attachments. I also fixed some issues determining the correct file extension for selected media that was causing upload to fail, as well as added better error handling for failed requests.

To test:

* Go to the Me page and tap "Submit feedback"
* Add several attachments (both images and videos) and verify they appear in the image pager
* Click **X** to remove an attachment and verify it's removed from the pager

Note that I added a new dependency: `io.coil-kt:coil-video`.  I try to avoid new dependencies, but since we already use Coil for images I felt okay with adding another Coil dependency (specifically for video thumbnails).

https://github.com/user-attachments/assets/180bcf81-2593-414a-bf3e-9fc19bc9085e
